Makefile

```diff
@@ -31,6 +31,9 @@ populate:
 run:
 	bin/app
 
+run-shib:
+	bin/app --auth shib
```

Comment:
I'm not a fan of you breaking the servers and the local developers (as github does not work with localhost:3000 whereas shibuqam does).
Morehover, the short term milestones targets the UQAM version.
Therefore, I, for one, thing that shib should be the default for now (we could change it latter)